src-opam/dockerfile_distro.ml

```diff
@@ -74,10 +80,20 @@ let distros = [
   `Ubuntu `V12_04; `Ubuntu `V14_04; `Ubuntu `V15_04; `Ubuntu `V15_10;
   `Ubuntu `V16_04; `Ubuntu `V16_10; `Ubuntu `V17_04; `Ubuntu `V17_10; `Ubuntu `V18_04; `Ubuntu `V18_10; `Ubuntu `V19_04; `Ubuntu `V19_10; `Ubuntu `V20_04; `Ubuntu `V20_10; `Ubuntu `V21_04;
   `Ubuntu `Latest; `Ubuntu `LTS;
-  `Cygwin `V20H2;
-  `Windows (`Mingw, `V1809); `Windows (`Mingw, `V1903); `Windows (`Mingw, `V1909); `Windows (`Mingw, `V2004); `Windows (`Mingw, `V20H2);
-  `Windows (`Msvc, `V1809); `Windows (`Msvc, `V1903); `Windows (`Msvc, `V1909); `Windows (`Msvc, `V2004); `Windows (`Msvc, `V20H2);
 ]
+let distros = List.fold_left (fun distros version ->
+                  `Cygwin version :: `Windows (`Mingw, version) :: `Windows (`Msvc, version) :: distros)
+                distros win10_releases
+
+(* https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/base-image-lifecycle *)
+let win10_release_status v ~servercore : status = match v with
```

Comment:
Server Core is an implementation detail - I think what's wanted here is the support channel. You could have a variant `type channel = SemiAnnual | LongTerm` or we could just use a boolean `~ltsc`.

Reply:
Changed that to explicitly list LTSC versions too. They're still supported, their semi-annual equivalent isn't and LTSC versions don't alias to semi-annual.